spec/features/users/user_profile_spec.rb

```diff
@@ -167,4 +168,17 @@
       end
     end
   end
+
+  context 'with a mobile device', js: true, driver: :headless_chrome_mobile do
+    before { sign_in_and_2fa_user }
+
+    it 'allows a user to navigate between pages' do
+      # Emulate reduced motion to avoid timing issues with mobile menu flyout animation
```

Comment:
Yeah, I was pretty happy about this, both from the perspective that this should be a feature we support anyways from an accessibility perspective, and also that it's a nice hack to avoid animation timing prolonging the feature specs.

And it also highlights the DevTools features we can use. In particular, I'm really interested in the [Accessibility APIs](https://chromedevtools.github.io/devtools-protocol/tot/Accessibility/) (see [related code comment](https://github.com/18F/identity-idp/pull/8327/files#diff-22994d177996832ecc4c7e61becfd3c600bef0add6ade54139be178f9dae3ac6R113-R123) in #8327).